apatheia.nimble

```diff
@@ -6,7 +6,10 @@ description   = "Async support for threading primitives"
 license       = "MIT"
 srcDir        = "src"
 
-
 # Dependencies
 
-requires "nim >= 1.6.18"
+requires "chronos >= 4.0.0"
+requires "threading"
+requires "taskpool"
```

Comment:
Nimble fails to resolve this dependency, is it used? If I comment it out all tests are passing.